# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/mac/MOXTextMarkerDelegate.h
# Commit: 1d37ebf3d20c
# Full Hash: 1d37ebf3d20ce8071c7482fbf7bcdc37834e950c
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2020-07-22 04:32:41
# Regressor Bug: 1653421
# File Overlap Count: 1
# Description:
#   Bug 1653421 - Part 2: Add attributed string for range getter. r=morgan
#   
#   This doesn't actually add any attributes yet.
#   VoiceOver uses this instead of the plain string equivalent.
#   
# ==============================================================================

diff -r daeda032926d -r 1d37ebf3d20c accessible/mac/MOXTextMarkerDelegate.h
--- a/accessible/mac/MOXTextMarkerDelegate.h	Tue Jul 21 23:02:53 2020 +0000
+++ b/accessible/mac/MOXTextMarkerDelegate.h	Tue Jul 21 23:02:55 2020 +0000
@@ -65,4 +65,7 @@
 // override
 - (id)moxPreviousTextMarkerForTextMarker:(id)textMarker;
 
+// override
+- (NSAttributedString*)moxAttributedStringForTextMarkerRange:(id)textMarkerRange;
+
 @end